client/webserver/site/src/js/charts.ts

```diff
@@ -454,11 +454,10 @@ export class DepthChart extends Chart {
   }
 
   // set sets the current data set and draws.
-  set (book: OrderBook, lotSize: number, rateStep: number, baseUnitInfo: UnitInfo, quoteUnitInfo: UnitInfo) {
+  set (book: OrderBook, lotSize: number, rateStepEnc: number, baseUnitInfo: UnitInfo, quoteUnitInfo: UnitInfo) {
     this.book = book
     this.lotSize = lotSize / baseUnitInfo.conventional.conversionFactor
-    const [qFactor, bFactor] = [quoteUnitInfo.conventional.conversionFactor, baseUnitInfo.conventional.conversionFactor]
-    this.rateStep = rateStep / RateEncodingFactor * qFactor / bFactor
+    this.rateStep = Doc.decodeRateStep(rateStepEnc, baseUnitInfo, quoteUnitInfo)
```

Comment:
A couple of things here. 1) it appears that this was wrong? See other comment on `bFactor / qFactor`. 2) `this.rateStep` should really be called `this.conventionalRateStep`, but not really your problem here, so whatever.

Reply:
> 1) it appears that this was wrong? See other comment on bFactor / qFactor

Yep, it contradicts this:
```
func ConventionalRateAlt(msgRate uint64, baseFactor, quoteFactor uint64) float64 {
	return float64(msgRate) / RateEncodingFactor * float64(baseFactor) / float64(quoteFactor)
}
```

> 2) this.rateStep should really be called this.conventionalRateStep

I would like to rename it, while we are at it, [done here](https://github.com/decred/dcrdex/pull/2278/commits/fc82f2b2fa856572a36d9ee64a0cd168bc78e0a6).